packages/devtools_app/lib/src/screens/network/network_controller.dart

```diff
@@ -57,6 +56,154 @@ class NetworkController extends DisposableController
     subscribeToFilterChanges();
   }
 
+  String? exportAsHarFile() {
+    httpRequests =
+        filteredData.value.whereType<DartIOHttpRequestData>().toList();
+
+    if (httpRequests!.isEmpty) {
+      debugPrint('No valid request data to export');
+      return '';
+    }
+
+    try {
+      if (httpRequests!.isNotEmpty) {
+        final har = {
+          'log': {
+            'version': '1.2',
+            'creator': {
+              'name': 'flutter_tool',
+              'version': '0.0.2',
```

Comment:
Where is this version from?

Reply:
http://www.softwareishard.com/blog/har-12-spec/
https://github.com/ahmadnassri/har-spec

Latest version is 1.2